src/i18n/fr.json

```diff
@@ -110,7 +110,10 @@
       "title": "Que voulez-vous ajouter"
     },
     "calendar": {
-      "title": "Quand cet événement ou cette activité a-t-il lieu ?",
+      "title": {
+        "events": "Quand cet événement ou cette activité a-t-il lieu ?",
+        "places": "Quand cet endroit ou cet emplacement est-il ouvert ?"
```

Comment:
```suggestion
        "events": "Quand cet événement ou cette activité a-t-il lieu?",
        "places": "Quand cet endroit ou cet emplacement est-il ouvert?"
```

Reply:
In French, exclamation points, question marks, colons, and semi-colons (all forms of "high" punctuation(Opens in a new window)) should always have a space preceding them.

Reply:
Wow 😲 Didn't know that